### PR TITLE
[DM-29082] Initial version

### PR DESCRIPTION
**Pull request by rra**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -1,29 +1,492 @@
...
+Usernames will be unique across the Science Platform and must satisfy the requirements given in DMTN-225_.
+If the user chooses, they can also specify a name and email address for the Science Platform to use in preference to the one released by their identity provider.
+The user will be required to verify that they can receive email at the email address they specify.
+
+At the conclusion of enrollment, the user will have a pending account on that Science Platform but will not yet have access.
+The user must then be approved for access to the Science Platform.
+That approval process will place the user in an appropriate access group for their data rights, as determined by the approver.
+This decision will usually based primarily on their institutional affiliation, but possibly based on other data gathered outside of the identity management system.
+Approvers may be Science Platform administrators or delegates who have the knowledge and authority to verify the data rights of a particular community of users.
```
Comment (frossie) [inline index.rst]:
```suggestion
Approvers will be project staff who have the knowledge and authority to verify the data rights of a particular community of users. Technical expertise in the identity management system is not required. 
```

File: index.rst [hunk 2]
```diff
@@ -1,29 +1,492 @@
...
+A new user of a general access deployment will go through an enrollment process.
+This process will gather the user's identity information as released by their federated identity provider (name, email, and institutional affiliation), and allow the user to select a username for use with the Science Platform.
+Usernames will be unique across the Science Platform and must satisfy the requirements given in DMTN-225_.
+If the user chooses, they can also specify a name and email address for the Science Platform to use in preference to the one released by their identity provider.
+The user will be required to verify that they can receive email at the email address they specify.
+
+At the conclusion of enrollment, the user will have a pending account on that Science Platform but will not yet have access.
+The user must then be approved for access to the Science Platform.
+That approval process will place the user in an appropriate access group for their data rights, as determined by the approver.
+This decision will usually based primarily on their institutional affiliation, but possibly based on other data gathered outside of the identity management system.
```
Comment (frossie) [inline index.rst]:
```suggestion
This decision will be taken according to observatory policy, but could be based on institutional affiliation and/or data obtained from outside the identity management system.
```

File: index.rst [hunk 3]
```diff
@@ -1,29 +1,492 @@
...
+-----------------------------
+
+Some Science Platform deployments run third-party services (Chronograf_, for example) that themselves want to do OpenID Connect authentication of the user.
+To support those services, the authentication service of the Science Platform is also an OpenID Connect provider.
+Such services can then point to the authentication service as the authentication provider, and those authentications will use the same source of identity as other authentications to the Science Platform.
+(This authentication is independent of any use of OpenID Connect by the authentication service to a federated or local identity provider external to the Science Platform, although the two authentications will be chained together when needed.)
+
+.. _Chronograf: https://www.influxdata.com/time-series-platform/chronograf/
+
+At present, OpenID Connect authentication used in this fashion does not do any access control.
```
Comment (frossie) [inline index.rst:401]: Is there an actual path to doing otherwise?

**Review by frossie — APPROVED** [review]

File: index.rst (hunk 3 above)
Comment (rra) [reply index.rst:401]: Yes: we could associate OpenID Connect clients with required scopes and refuse to authenticate the user if they're missing that scope. There will be more discussion of this in SQR-069.

**Pull request merged**